membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala

```diff
@@ -125,15 +97,15 @@ object Product {
   type Contribution = Contribution.type
 }
 
-sealed trait Benefit {
+// Benefit is the catalog charge level ProductType__c
+sealed trait ProductRatePlanChargeProductType {
```

Comment:
there are two product type fields in zuora, one on the product and one on the product rate plan charge. The latter used to be treated as the "benefit" but now it's got a more accurate name (if a little verbose)